src/app/contenido-interactivo/revision-content/revision-content.component.html

```diff
@@ -10,13 +10,26 @@
                 <mat-list>
                     <mat-list-item cols="2" *ngFor="let q of dataQuestions">
                         <h4 mat-line>{{q.question}}</h4>
-                        <p mat-line><b>Respuesta correcta:</b></p>
+                        <div mat-line style=" width: 100%">
```

Comment:
Please move this inline style to the style sheet

Reply:
Listo, hecho :+1: